lib/plain/entities/function-log.ts

```diff
@@ -0,0 +1,55 @@
+import type {
+  CollectionProp,
+  GetFunctionLogParams,
+  GetAllFunctionLogParams,
+} from '../../common-types'
+import type { FunctionLogProps } from '../../entities/function-log'
+import type { OptionalDefaults } from '../wrappers/wrap'
+
+export type FunctionLogPlainClientAPI = {
+  /**
+   * Fetches the specified FunctionLog
+   * @param spaceId
+   * @param environmentId
+   * @param appInstallationId
+   * @param functionId
+   * @param logId
```

Comment:
I forgot that the proper usage of the `@param` directive is actually for each argument to a function, so all of these functions actually only have a single param, i.e. ...
```suggestion
   * @param {OptionalDefaults<GetSpaceEnvironmentParams>} params
```